parse/grammar/KuneiformParser.g4

```diff
@@ -184,13 +216,58 @@ sql:
 
 sql_statement:
     (WITH RECURSIVE? common_table_expression (COMMA common_table_expression)*)?
-    (select_statement | update_statement | insert_statement | delete_statement)
+    (create_table_statement | alter_table_statement| create_index_statement | drop_index_statement // ddl
```

Comment:
This seems incorrect. DDL shouldn't have CTEs. For example, I cant have:

```sql
WITH my_cte AS (
-- ...
)
CREATE TABLE ...
```

These should instead be separate clauses

Reply:
oh  I wanted to make a quick pass first then create another statement, totally forget about it.  A new `ddl_statement` sounds good?